--- origsrc/libroadnav-0.19/Makefile.am	2007-10-08 20:44:26.000000000 -0500
+++ src/libroadnav-0.19/Makefile.am	2008-07-01 01:00:04.312500000 -0500
@@ -1,7 +1,6 @@
 SUBDIRS = src samples win32 include data
 
-libroadnavdocdir = ${prefix}/share/doc/libroadnav
-libroadnavdoc_DATA = \
+noinst_DATA = \
 	README\
 	COPYING\
 	AUTHORS\
@@ -10,7 +9,7 @@
 	NEWS\
 	TODO
 
-EXTRA_DIST = $(libroadnavdoc_DATA) autogen.sh
+EXTRA_DIST = $(noinst_DATA) autogen.sh
 
 # Copy all the spec files. Of cource, only one is actually used.
 dist-hook:
--- origsrc/libroadnav-0.19/samples/wx/minimal/Makefile.am	2007-10-08 20:44:26.000000000 -0500
+++ src/libroadnav-0.19/samples/wx/minimal/Makefile.am	2008-06-30 23:18:11.515625000 -0500
@@ -1,11 +1,11 @@
 INCLUDES = \
 	$(WX_CXXFLAGS)\
-	-I../../../include
+	-I$(top_srcdir)/include
 
 AM_CXXFLAGS =\
 	 -Wall\
 	 -O2\
-	 -I../../../src
+	 -I$(top_srcdir)/src
 
 noinst_PROGRAMS = minimal
 
--- origsrc/libroadnav-0.19/samples/wx/savemap/Makefile.am	2007-10-08 20:44:26.000000000 -0500
+++ src/libroadnav-0.19/samples/wx/savemap/Makefile.am	2008-06-30 23:18:29.484375000 -0500
@@ -1,11 +1,11 @@
 INCLUDES = \
 	$(WX_CXXFLAGS)\
-	-I../../../include
+	-I$(top_srcdir)/include
 
 AM_CXXFLAGS =\
 	 -Wall\
 	 -O2\
-	 -I../../../src
+	 -I$(top_srcdir)/src
 
 noinst_PROGRAMS = savemap
 
--- origsrc/libroadnav-0.19/src/Makefile.am	2007-10-08 20:44:26.000000000 -0500
+++ src/libroadnav-0.19/src/Makefile.am	2008-06-30 23:51:18.359375000 -0500
@@ -1,6 +1,7 @@
 INCLUDES = \
 	$(WX_CXXFLAGS)\
-	-I../include/libroadnav
+	-DWXMAKINGDLL_HTTPENGINE \
+	-I$(top_srcdir)/include/libroadnav
 
 AM_CXXFLAGS =\
 	 -Wall\
